pom.xml

```diff
@@ -150,9 +150,9 @@
   </dependencyManagement>
   <dependencies>
     <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-      <version>3.11</version>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>commons-lang3-api</artifactId>
+      <version>3.12.0-36.vd97de6465d5b_</version>
```

Comment:
Isn't the version already set by the BOM?

```suggestion
```

Reply:
no. it's not